app/src/test/java/ayds/winchester/songinfo/otherinfo/model/repository/RepositoryTest.kt

```diff
@@ -0,0 +1,60 @@
+package ayds.winchester.songinfo.otherinfo.model.repository
+
+import ayds.winchester.songinfo.moredetails.model.entities.Card
+import ayds.winchester.songinfo.moredetails.model.entities.CardDescription
+import ayds.winchester.songinfo.moredetails.model.entities.Source
+import ayds.winchester.songinfo.moredetails.model.repository.CardRepositoryImpl
+import ayds.winchester.songinfo.moredetails.model.repository.external.BrokerService
+import ayds.winchester.songinfo.moredetails.model.repository.local.LocalRepository
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.junit.Assert
+import org.junit.Test
+
+class RepositoryTest {
+    private val localRepository: LocalRepository = mockk(relaxUnitFun = true)
+    private val broker : BrokerService = mockk(relaxUnitFun = true)
+
+    private val descriptionRepository: CardRepositoryImpl by lazy {
+        CardRepositoryImpl(localRepository, broker)
+    }
+
+    @Test
+    fun `given existing cards should return cards and mark it as local`() {
+        var cards: MutableList<Card> = mutableListOf()
+        val card : Card = CardDescription("", "1", Source.WIKIPEDIA, "" )
+        cards.add(card)
+        every { localRepository.getArtistDescription("artist") } returns cards
+
+        val result = descriptionRepository.getCards("artist")
+
+        Assert.assertEquals(cards, result)
+        Assert.assertTrue(cards.last().isLocallyStored)
```

Comment:
Debería checkearse que todas las cards se marcaron, de manera más genérica
```suggestion
        Assert.assertTrue(cards.all { it.isLocallyStored })
```